pkg/ring/kv/kv_test.go

```diff
@@ -197,6 +201,7 @@ func TestWatchPrefix(t *testing.T) {
 				}
 				require.NoError(t, err)
 			}
+			fmt.Println("Generator finished in", time.Since(start))
```

Comment:
If you need to keep these `fmt.Println()`, what do you think about `t.Log()` instead? Otherwise I would drop them.

Reply:
> is the snappy upgrade worth a CHANGELOG entry about what it fixes?

It only fixes arm64 code on (still unreleased) go1.16. Given that, I don't think it's worth the changelog entry.